### ISSUE TITLE
[A11y] Desktop/Mobile Work Show Page-Single Item: UV iframe

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1982]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Work Show

### Admin user?

No

### Issue Description

Work Show Page-Single Item: https://staging.oregondigital.org/concern/images/1g05fb620?locale=en

The UV iframe has the following failed instance:
link-name: Ensures links have discernible text

Path: 
iframe[aria-label="image\ view"];#top

Snippet:
`<a href="/" id="top"></a>`

### Suggested Fix

Fix the following:

- Element is in tab order and does not have accessible text
- Fix ONE of the following:
- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

### Can anybody QA or does this need an expert?

A11y familiarity recommended

**Comment by azle** [comment]
Updated 12/20/2024

Prior issue fixed however a new one dealing witth list.

Snippet:
<ul class="tabular">

How to fix:

Fix the following:
List element has direct children that are not allowed: br
